tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -84,7 +84,11 @@ protected function tearDown()
     {
         chdir($this->workingDirectory);
         $filesystem = new Filesystem();
-        $filesystem->remove($this->tempDir);
+        if (!defined('PHP_WINDOWS_VERSION_BUILD')) {
+            $filesystem->remove($this->tempDir);
+        } else {
+            exec(sprintf("rd /s /q %s", escapeshellarg($this->tempDir)));
```

Comment:
I'm gonna check but IMO it is something it should handle. In any case it's not gonna be fixed immediately so let's go with it for now